src/api/Feed.js

```diff
@@ -504,6 +505,19 @@ class Feed extends Base {
             },
             errorCallback: handleError,
         });
+
+        const params = queryString.parse(location.search);
```

Comment:
Can we retrieve this information in a component and pass it to this call? Doing so in the data access layer isn't ideal.